build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs

```diff
@@ -42,9 +42,7 @@ public class SharedFrameworkPublisher
             _corehostLockedDirectory = corehostLockedDirectory;
             _corehostLatestDirectory = corehostLatestDirectory;
             _corehostPackageSource = corehostPackageSource;
-            _crossgenUtil = (sharedFrameworkRid == "win10-arm64") ?
-                new Crossgen(DependencyVersions.CoreCLRVersion_Latest, DependencyVersions.JitVersion_Latest, sharedFrameworkRid) : 
-                new Crossgen(DependencyVersions.CoreCLRVersion, DependencyVersions.JitVersion); 
+            _crossgenUtil = new Crossgen(DependencyVersions.CoreCLRVersion, DependencyVersions.JitVersion, sharedFrameworkRid == "win10-arm64" ? sharedFrameworkRid : null);
```

Comment:
It's invoked for crossgen later down the line. The RID is so we get the right version since the folders are different for arm64 (e.g. tools/x64_arm64/crossgen.exe instead of just tools/crossgen.exe)